README.md

```diff
@@ -140,7 +141,7 @@ response = gpt_4o_mini.generate_text(
 Harnessing the power of Python, you can easily create your own plugins to add additional functionality to your conversations:
 
 ```python
-class SimpleMemoryPlugin:
+class SimpleMemoryPlugin(sm.BasePlugin):
```

Comment:
i also want to keep the simple memory plugin just a base class, to show that it's allowed.

Reply:
done 👍🏻